final?

FelStroz opened the pull request.

tony-matheus committed:
Update >

FelStroz committed:
adicionando novas funções dos operadores

FelStroz committed:
adicionando operador de concatenação

FelStroz committed:
adicionando comentários para melhor entendimento do código

FelStroz committed:
adicionando operadores lógicos

FelStroz committed:
adicionando a funcionalidade de CIN

tony-matheus reviewed (commented):
Altera esses dois comentários

FelStroz committed:
arrumando operadores que estavam com codigo duplicado

FelStroz committed:
Update String.h

FelStroz committed:
Arrumando arquivo de execução do programa

adicionando novas funcionalidades de concatenação e ajustes de saidas para ficar mais intuitivo

FelStroz committed:
adicionando funcionalidade de concatenação (comentada)

FelStroz committed:
Adicionando funcionalidade de concatenação

faltando apenas a concatenação de "asd"+"asdasd"

FelStroz committed:
arrumando a concatenação no programa principal
